src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1842,10 +1868,75 @@ namespace rascal {
                   l_block_idx += l_block_size;
                 }  // for (angular_l)
               }    // for cartesian_idx
-            }
-          }  // if (IsHalfNL)
-        }    // if (this->compute_gradients)
-      }      // for (neigh : center)
+            }      // if (is_center_atom)
+          }        // if (IsHalfNL)
```

Comment:
The closing if-statement comments no longer match the opening conditions, please update

Reply:
it does to me. Could you check again or be more precise ?

Reply:
Oh, the `IsHalfNL` check didn't actually disappear, it just got moved. Sorry, it didn't show up properly in the diff view. But why is the `IsHalfNL` test still here, when you said that the option now marked as incompatible with this representation?